_posts/2017-10-04-Good-Bad-And-Ugly.md

```diff
@@ -14,49 +14,49 @@ image: 'assets/images/jsynowiec.jpg'
 
-I settled on a workflow that I have found on some ServiceNow [forum](https://snstud.io/using-git-servicenow/), saying that to merge branches you should stash often, then switch branch and unstash. I have not been able to find any official feedback on using git integration for actual version control. Instead, ServiceNow recommend using it for managing versions of the application and easy installing application on developer instances. That means people are wanting to use git as a version control system, but they experience similar problems as I do. Accepting the fact that I have to do a weirdest workaround with VCS system I have ever seen, I was happy to say that it worked for me.
+I settled on a workflow that I have found on some ServiceNow [forum](https://snstud.io/using-git-servicenow/), saying that to merge branches you should stash often, then switch branch and unstash. I have not been able to find any official feedback on using git integration for actual version control. Instead, ServiceNow recommends using it for managing versions of the application and easy installing application on developer instances. That means people want to use git as a version control system, but they experience similar problems as I do. Accepting the fact that I have to do the weirdest workaround with VCS system I have ever seen, I was happy to say that it worked for me.
 
 For a few days.
 
-One day, after stashing some changes and trying to unstash them on a different branch, unstashing failed. Up to this day I do not know why did it fail. It said that I have unresoled conflicts in files, and then gave me an empty list of files that I should resolve confilts in. I was annoyed not only because of it not working properly, but because of me loosing some work that I have been doing over some time.
+One day, after stashing some changes and trying to unstash them on a different branch, unstashing failed. Up to this day I do not know why did it fail. It said that I have unresoled conflicts in files, and then gave me an empty list of files that I should resolve confilts in. I was annoyed not only because of it not working properly, but because of me losing some work that I have been doing over some time.
```

Comment:
One day, after stashing some changes and trying to unstash them on a different branch, unstashing failed. Up to this day I do not know why it failed. It said that I have unresolved conflicts in files, and then gave me an empty list of files that I should resolve conflicts within. I was annoyed not only because of it not working properly, but because of me losing some work that I have been doing over some time.